.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Thank you for adding the test.md file in the gitignore. you are also required to add node_modules/ folder to the gitignore file